apps/gateway/src/routes/api/v1/projects/:projectId/commits/:commitUuid/documents/handlers/_shared.ts

```diff
@@ -37,7 +36,7 @@ export const getData = async ({
   const document = await docsScope
     .getDocumentByPath({
       commit,
-      path: toDocumentPath(documentPath),
+      path: documentPath,
```

Comment:
This is wrong no? The API should always be sent the full document path so there is no reason to prefix with `/`

Reply:
well this is shared with get endpoint which can't include the leading / because document path being a url parameter.

Reply:
So it would have to be unshared 😂